py/bitbox02/bitbox02/bitbox02/bootloader.py

```diff
@@ -94,6 +94,15 @@ def versions(self) -> typing.Tuple[int, int]:
         firmware_v, signing_pubkeys_v = struct.unpack("<II", response[:8])
         return firmware_v, signing_pubkeys_v
 
+    def hardware(self) -> str:
```

Comment:
Please no stringly types. An enum like this would be better imho:

https://github.com/BitBoxSwiss/bitbox02-firmware/blob/master/py/bitbox02/bitbox02/communication/bitbox_api_protocol.py#L187

Also this should return the ATECC variant if the bootloader version does not support this endpoint. Right now this call fails for old bootloaders.